server/kit/gae_log.go

```diff
@@ -0,0 +1,168 @@
+package kit
+
+import (
+	"context"
+	"encoding"
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"strings"
+
+	"cloud.google.com/go/logging"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/pkg/errors"
+	"google.golang.org/genproto/googleapis/api/monitoredres"
+)
+
+type gaeLogger struct {
+	project string
+	monRes  *monitoredres.MonitoredResource
+	lc      *logging.Client
+	lgr     *logging.Logger
+}
+
+func newAppEngineLogger(ctx context.Context, projectID, service, version string) (log.Logger, error) {
+	client, err := logging.NewClient(ctx, fmt.Sprintf("projects/%s", projectID))
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to initiate stackdriver log client")
+	}
+	return gaeLogger{
+		lc:      client,
+		lgr:     client.Logger("app_logs"),
+		project: projectID,
+		monRes: &monitoredres.MonitoredResource{
+			Labels: map[string]string{
+				"module_id":  service,
+				"project_id": projectID,
+				"version_id": version,
+				"zone":       "us6",
```

Comment:
Is this intended to be hard-coded?

Reply:
oh! good catch! was a test as I was getting this to work.